back/spec/acceptance/ideas_spec.rb

```diff
@@ -395,13 +395,27 @@
 
       context 'idea authored by user' do
         let!(:idea) do
-          create(:idea, project: phase.project, phases: [phase], creation_phase: phase, author: @user, publication_status: 'draft')
+          create(:native_survey_response, project: phase.project, author: @user, publication_status: 'draft', custom_field_values: { 'field' => 'value' })
         end
```

Comment:
Was this supposed to be changed from an `idea` to a `native_survey_response`?

Reply:
Yes, doesn't make a difference to the test really but the phase is a native survey phase so made sense to use the  native_survey_response factory